lib/db/upgrade.php

```diff
@@ -3907,6 +3907,40 @@ function xmldb_main_upgrade($oldversion) {
         upgrade_main_savepoint(true, 2022020200.01);
     }
 
+    // Unique indexes for question_set_references.
+    if ($oldversion < 2022020500.00) {
```

Comment:
it can be done in a same chunk, please merge them to one.

Reply:
Done.